devicetypes/smartthings/zigbee-lock.src/zigbee-lock.groovy

```diff
@@ -500,6 +520,11 @@ private def parseAttributeResponse(String description) {
 	} else if (clusterInt == CLUSTER_DOORLOCK && attrInt == DOORLOCK_ATTR_NUM_PIN_USERS && descMap.value) {
 		def maxCodes = Integer.parseInt(descMap.value, 16)
 		responseMap = [name: "maxCodes", value: maxCodes, descriptionText: "Maximum Number of user codes supported is ${maxCodes}", displayed: false]
+	} else if (clusterInt == CLUSTER_IAS_ZONE && attrInt == IAS_ATTR_ZONE_STATUS && descMap.value) {
+		def descValue = Long.parseLong(descMap.value,16)
```

Comment:
Better to use `def zs = new ZoneStatus(zigbee.convertToInt(descMap.value, 16))` and then you can use `zs.isBatterySet()`

Reply:
It should be ok, if lock would send value in "int" type. What lock sends must be parse as "long", because of exception that I get. And ZoneStatus has only constuctor with "int" argument. I would leave this as it is.

Reply:
What exception do you get?

Reply:
java.lang.NumberFormatException: For input string: "24fd5b000002dac8" @line 524 (parseAttributeResponse) when I try to parse value as Integer.

Reply:
So I'm really confused.  Does this device actually use the CIE address to report battery? I really doubt that.  I think the comment above about attribute `0x0010` being CIE address and zone status being `0x0002` it was meant for you to change the value to `0x0002` not for you to change the name to CIE address

Reply:
I think it is my mistake, after configuration I got:

description map:- [raw:8109010500161000F0C8DA0200005BFD24, dni:8109, endpoint:01, cluster:0500, size:22, attrId:0010, result:success, encoding:f0, value:24fd5b000002dac8, isValidForDataType:true, clusterInt:1280, attrInt:16]

And there was diffrence only with value that had 2 states:  24fd5b000002dac8 or 00000000000000.

Reply:
I fixed my mistake.